src/SiddhiB/Asgn2ArrayFindUniqueNumber.java

```diff
@@ -0,0 +1,36 @@
+package SiddhiB;
+
+public class Asgn2ArrayFindUniqueNumber
+{
+	static void uniqueNumber()
+	{
+		int [] num1 = {10,13,11,19,13};
+		int [] num2	= {18,11,10,29,11};
+		int [] outArray = new int [num1.length+num2.length];
```

Comment:
As per the my debug it print only 10,13,11,19
Please write SOP at appropriate place.